packages/scheduler/src/Scheduler.js

```diff
@@ -449,39 +449,42 @@ var shouldYieldToHost;
 
 if (typeof window !== 'undefined' && window._schedMock) {
   // Dynamic injection, only for testing purposes.
-  var impl = window._schedMock;
-  requestHostCallback = impl[0];
-  cancelHostCallback = impl[1];
-  shouldYieldToHost = impl[2];
+  var windowImpl = window._schedMock;
+  requestHostCallback = windowImpl[0];
+  cancelHostCallback = windowImpl[1];
+  shouldYieldToHost = windowImpl[2];
+  getCurrentTime = windowImpl[3];
+} else if (typeof global !== 'undefined' && global._schedMock) {
+  // Dynamic injection, only for testing purposes.
+  var globalImpl = global._schedMock;
+  requestHostCallback = globalImpl[0];
+  cancelHostCallback = globalImpl[1];
+  shouldYieldToHost = globalImpl[2];
+  getCurrentTime = globalImpl[3];
```

Comment:
This is a little awkward. I could combine these branches?

Reply:
Let's combine.

Reply:
Sure, combined in 9a19b875d